amethyst_animation/src/resources.rs

```diff
@@ -13,13 +14,16 @@ pub trait AnimationSampling: Send + Sync + 'static {
     /// The interpolation primitive
     type Primitive: InterpolationPrimitive + Clone + Copy + Send + Sync + 'static;
     /// The channel type
-    type Channel: Clone + Hash + Eq + Send + Sync + 'static;
+    type Channel: Debug + Clone + Hash + Eq + Send + Sync + 'static;
```

Comment:
Debug bound may not be necessary.

Reply:
I could put it here or on every place where the channel is used, it's required for the use in `AnimationCommand`